Components/FlightControl/Inc/TimerTransitions.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ ******************************************************************************
+ * File Name          : TimerTransitions.cpp
+ * Description        : Primary Watchdog task, default task for the system.
```

Comment:
Change this desc

Reply:
lol. the copy paste catching me off guard

Reply:
please change 😅

Reply:
thank you so much for your feedback
it helps me never slack
kind like how things were back in the days of mack